engine/src/test/java/org/hibernate/validator/test/internal/xml/XmlParsingTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.test.internal.xml;
+
+import static org.hibernate.validator.testutils.ValidatorUtil.getConfiguration;
+import static org.testng.Assert.assertEquals;
+
+import java.util.Set;
+
+import javax.validation.Validator;
+import javax.validation.metadata.BeanDescriptor;
+import javax.validation.metadata.ConstraintDescriptor;
+
+import org.assertj.core.api.Assertions;
+import org.hibernate.validator.internal.metadata.descriptor.ConstraintDescriptorImpl;
+import org.hibernate.validator.testutil.TestForIssue;
+import org.junit.Test;
```

Comment:
HV uses TestNG. I'll fix it. Should we add a CheckStyle check of sorts to prevent importing the JUnit annotations?